src/db/infusion/model.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const infusionSchema = new Schema({
+  startVolume: { type: String },
+  stopVolume: { type: String },
```

Comment:
Please, let's make the start and stop volume of `number` type. That would make it easier to perform calculations with them.

Reply:
fixed

Reply:
fixed